tests/testthat/test-step-subset.R

```diff
@@ -38,6 +38,11 @@ test_that("simple calls generate expected translations", {
     expr(DT[, .(a = x, y)])
   )
 
+  expect_equal(
```

Comment:
I think `relocate()` is complicated enough that it should have it's own test case — it needs to have an example of exactly one of `.before` and `.after`, neither of them, and both of them. I'd also look at https://github.com/tidyverse/dplyr/blob/master/tests/testthat/test-relocate.R to see if there are other tricky cases we should also test for here.

Reply:
I've added tests based off dplyr, as suggested. I also added one to ensure relocation does not change the grouping variables.